packages/generator/templates/app/_forms/hookform/LabeledTextField.tsx

```diff
@@ -1,5 +1,5 @@
-import { forwardRef, PropsWithoutRef } from "react"
-import { useFormContext } from "react-hook-form"
+import {forwardRef, PropsWithoutRef} from "react"
+import {useFormContext} from "react-hook-form"
 
 export interface LabeledTextFieldProps extends PropsWithoutRef<JSX.IntrinsicElements["input"]> {
```

Comment:
Since RHF v7 supports strongly typed `name` fields, ideally we fix our TS here to support that too. This current setup drops that benefit. But it could be take up later if you are not sure how to do it (I'm not sure if or how to do it either)

Reply:
I don't know how to do it with the current setup. Mainly because we are passing the schema to the Form instead of the TextField